first draft

Adimote opened the pull request:
There's a lot of content here, I'd like to focus people's attention on the content of the form, as it's more important that we get that bit right.

Adimote committed:
first draft

Adimote committed:
fix font choice, and fix the resultant widow

Adimote committed:
add readme

anton-firmware commented on form.tex:
```diff
@@ -0,0 +1,92 @@
+\vspace{0.5cm}
+\textbf{\large Privacy Policy}\\
+
+Once signed, this form will be stored securely and encrypted at the University of Southampton for the entire time any recordings made at \competition is used. All recordings will be openly available to the internet. You have the right to request for the deletion of any personal data (including images) as per the EU General Data Protection Regulation (GDPR).
```
> thampton for the entire time any recordings made at \competition **is** used

Recordings made **are** used ???

Adimote replied:
fixed in 16667cd261ad94181a1ddf398eb74bf73aa57f37

anton-firmware commented on form.tex:
```diff
@@ -0,0 +1,92 @@
+\end{minipage}
+\vspace{1cm}
+
+\textbf{This is not a ticket.}\\\\ However, this form is required for entry into the \competition competition.\\
```
> \textbf{This is not a ticket.

#ExtremePedantry: Put **!** instead of a .

Adimote replied:
#TooExtremePedantryForMeToBother

anton-firmware reviewed (dismissed):
just some minor things :)

Adimote committed:
s/is/are

PeterJCLaw commented on Makefile:
```diff
@@ -0,0 +1,34 @@
+	while true; do \
+		inotifywait --event CLOSE_WRITE -q -r *.tex; \
+			make; \
+	make servercount; \
```
What does `make servercount` do?

Adimote replied:
oops, copied this from a different script

PeterJCLaw commented on form.tex:
```diff
@@ -0,0 +1,92 @@
+
+Individual for media consent to be granted:\\
+\begin{Form}
+Full Name: \vbox{\hbox{\TextField[width=12.7cm]{\null}}\kern2pt\hrule}\\
```
In GNOME Document Viewer 3.18.2 on Ubuntu I get bright red boxes here in addition to the underlines. Is that expected?

I don't get the boxes in Firefox.

RealOrangeOne replied:
Using bright red text is the default for `hyperref`, however, I tried turning it off, and nothing happened. https://tex.stackexchange.com/questions/823/remove-ugly-borders-around-clickable-cross-references-and-hyperlinks

trickeydan replied:
I have found that https://tex.stackexchange.com/questions/327487/latex-fillable-form-remove-redboxes fixes this issue for me.

PeterJCLaw commented on form.tex:
```diff
@@ -0,0 +1,92 @@
+\begin{itemize}
+    \item The University of Southampton
+    \item SourceBots
+    \item Current or future sponsors of SourceBots
```
Why would "current or future sponsors" be recording people? Are we expecting to invite them to the competition and that they'll do their own recordings?

I don't think we need this.

PeterJCLaw replied:
Related: the following paragraph says that the recordings can be used for promoting (any) events of any of the parties. In theory this means that our recordings could be used to promote a completely unrelated event run by one of our sponsors. I'm not sure we want that, nor that that's reasonable to ask for.

PeterJCLaw commented on form.tex:
```diff
@@ -0,0 +1,92 @@
+Your recordings may be used in publications (including but not limited to; press releases, websites, blog posts, livestreams, videos, brochures, and posters) for the purpose of publicising/promoting the event and future events held by the above parties.
+
+\vspace{0.5cm}
+\textbf{\large Privacy Policy}\\
```
Maybe pull this section to the top so that it's clear that the privacy policy is only this paragraph (and doesn't extend to the following paragraphs)?

Adimote replied:
fixed in 3ea32829b9d7ebe3ff152e98b2a499d39c3cbc91

anton-firmware commented on README.md:
```diff
@@ -0,0 +1,14 @@
+
+## Making a form
+
+You must have `pdflatex`, `make`, and either `svg2pdf` or `inkscape` installed.
```
you could also suggest a normal GUI tex editor ??? :) Like TeXStudio or TeXMaker ????

RealOrangeOne replied:
I'm not sure it's compatible. The build process builds SVGs into PDF assets, which I doubt those editors will do automatically. Whilst I don't like this custom build process, it works, well, and is stable.

anton-firmware commented on form.tex:
```diff
@@ -0,0 +1,92 @@
+\vspace{0.5cm}
+\textbf{\large Privacy Policy}\\
+
+Once signed, this form will be stored securely and encrypted at the University of Southampton for the entire time any recordings made at \competition are used. All recordings will be openly available to the internet. You have the right to request for the deletion of any personal data (including images) as per the EU General Data Protection Regulation (GDPR).
```
is it an idea to suggest who to contact? Or is there no point? :)

Adimote replied:
Currently we don't have a good email for this, later on we'll have some nice way to contact us on our website, but in the mean time I just decided to leave it ambiguous on who to contact

anton-firmware reviewed (commented):
just some comments, I haven't left any specific points to change as it's up to you if you want to make these...

PeterJCLaw commented on form.tex:
```diff
@@ -0,0 +1,92 @@
+\vspace{0.5cm}
+\textbf{\large Privacy Policy}\\
+
+Once signed, this form will be stored securely and encrypted at the University of Southampton for the entire time any recordings made at \competition are used. All recordings will be openly available to the internet. You have the right to request for the deletion of any personal data (including images) as per the EU General Data Protection Regulation (GDPR).
```
While the GDPR comment is true, I'm not sure how to apply it here. On the one hand an image _of a person_ is clearly of that person. However, unless the image is tagged as being of that person then it's just an image and the subject could be anyone.

From a practical perspective we don't need to tag the images, nor do we really want to. Yet without that, finding and removing the images of a particular person would be quite burdensome.

Should we clarify that we don't intend to tag the images with anything more precise than, say, the team or school name? And then only when they're the intended subject of the image? (Obviously we'd want to be clear that this is something we _may_ do, rather than something we _will_ do)

Adimote replied:
I've been led to believe that a photo contains someone's personal information as long as their face is recognisable in it, not just if they're the subject of the photo.

Therefore I don't think tagging the photo with someone's name changes the situation, as there still exists personal information which must be removed if they ask. In that situation, it would be a simple case of scrolling through 500-odd photos, looking for someone's face in them, which humans are pretty fast at doing.

PeterJCLaw commented on form.tex:
```diff
@@ -0,0 +1,92 @@
+\vspace{0.5cm}
+\textbf{\large Privacy Policy}\\
+
+Once signed, this form will be stored securely and encrypted at the University of Southampton for the entire time any recordings made at \competition are used. All recordings will be openly available to the internet. You have the right to request for the deletion of any personal data (including images) as per the EU General Data Protection Regulation (GDPR).
```
I'm also not sure how GDPR interacts with contracts in the general case. The idea that a party to a contract could nullify it by requesting that it be destroyed under GDPR is obviously silly as it would turn the GDPR into a super-get-out for everything. As a result, offering to destroy these forms in response to a right-to-be-forgotten request might not be a useful or accurate thing to do.

Adimote replied:
I don't believe these forms are contracts, they are more requests for consent to be recorded, and their data stored. 

The paragraph is currently worded in a way that it doesn't explicitly mention that the forms are  personal data that can be deleted. Sure, they can request for the deletion of the form, and at that point we can decide whether it should actually be deleted.

However, assuming these are contracts, and therefore we could refuse to delete them. People could still request for all non-essential personal data to be deleted, which would be all recordings we have of them, which would make these contracts pointless to keep anyway.

PeterJCLaw commented on form.tex:
```diff
@@ -0,0 +1,92 @@
+\vspace{0.5cm}
+\textbf{\large Privacy Policy}\\
+
+Once signed, this form will be stored securely and encrypted at the University of Southampton for the entire time any recordings made at \competition are used. All recordings will be openly available to the internet. You have the right to request for the deletion of any personal data (including images) as per the EU General Data Protection Regulation (GDPR).
```
Do we need to mention the Data Protection Act here, or does the GDPR end up replacing it?

Adimote replied:
according to https://www.itgovernance.co.uk/data-protection:

> The DPA will soon be superseded by the EU General Data Protection Regulation (GDPR)

PeterJCLaw replied:
:+1:, however _soon_. Note that the GDPR is not yet in force, so the DPA is the current regulation.

PeterJCLaw commented on form.tex:
```diff
@@ -0,0 +1,92 @@
+\vspace{0.5cm}
+\textbf{\large Privacy Policy}\\
+
+Once signed, this form will be stored securely and encrypted at the University of Southampton for the entire time any recordings made at \competition are used. All recordings will be openly available to the internet. You have the right to request for the deletion of any personal data (including images) as per the EU General Data Protection Regulation (GDPR).
```
Why are we encrypting the forms? Is the plan to scan and encrypt them, then destroy the originals?

I think it would be more usual to just keep the paper copies in a lock-box.

Adimote replied:
I'm assuming the majority of teachers will email us the scanned copies, I don't really want to print them just so we have all of them on paper. Is there any specific reason why it's better to keep them in physical form?

Adimote replied:
I've changed this bit quite dramatically in 1fbd26d448c6cd10c34b4633684ead6de2e73614 (removing mentions of encryption, and said it won't necessarily be in the uni)

PeterJCLaw commented on form.tex:
```diff
@@ -0,0 +1,92 @@
+\begin{center}Please sign ONE of the two sections below:\end{center}
+
+\begin{minipage}[t]{0.48\textwidth}
+\textbf{For those aged 12 or older:}
```
How many under-16s do we expect to turn up? Of those, how many without a guardian?

If not many, then I'd be in favour of bounding this at 16 rather than 12. Even just from practical perspective, age 12 is quite young to be signing stuff.

I picked 16 since most (all?) competitors will be 16 or over.

As a separate reference point, for scouts (age 10-14) I'd definitely expect to get the parent's consent, though would of course still speak to the young people as well.

Adimote replied:
Okay, that's fair. Changed the age to 16 in b3c6c031187a6a8c277e151fe586cdf4b3435b31

PeterJCLaw commented on form.tex:
```diff
@@ -0,0 +1,92 @@
+    \item SourceBots
+    \item Current or future sponsors of SourceBots
+\end{itemize}
+Your recordings may be used in publications (including but not limited to; press releases, websites, blog posts, livestreams, videos, brochures, and posters) for the purpose of publicising/promoting the event and future events held by the above parties.
```
"Your recordings" reads as "recordings belonging to you", which isn't what we want to mean here. Perhaps "The recordings"?

Adimote replied:
Fixed in fc455f674de02a847cfb7ec62c3fcfb4cecb1e54

PeterJCLaw commented on form.tex:
```diff
@@ -0,0 +1,92 @@
+    \item SourceBots
+    \item Current or future sponsors of SourceBots
+\end{itemize}
+Your recordings may be used in publications (including but not limited to; press releases, websites, blog posts, livestreams, videos, brochures, and posters) for the purpose of publicising/promoting the event and future events held by the above parties.
```
Maybe expand the last sentence to:
> ... for the purposes of publicising or promoting the current or future activities of the above parties.

Adimote replied:
fixed in 31a1beeb658162595c8dd179295fbb4c00e4adc1

PeterJCLaw commented on form.tex:
```diff
@@ -0,0 +1,92 @@
+\textbf{This is not a ticket.}\\\\ However, this form is required for entry into the \competition competition.\\
+\vspace{0.3cm}
+
+\textbf{\large Media Consent}\\
```
We should clarify that this consent applies only to recordings at the current event. This ends up being implied by the privacy policy at the moment, but I think we should be precise here and then be less specific in the privacy policy.

Adimote replied:
fixed in 6ff922efd07757169a82cb4d3bc272783c720d49

PeterJCLaw commented on form.tex:
```diff
@@ -0,0 +1,92 @@
+\vspace{0.5cm}
+\textbf{\large Privacy Policy}\\
+
+Once signed, this form will be stored securely and encrypted at the University of Southampton for the entire time any recordings made at \competition are used. All recordings will be openly available to the internet. You have the right to request for the deletion of any personal data (including images) as per the EU General Data Protection Regulation (GDPR).
```
> All recordings will be openly available to the internet.

I appreciate what this is saying, though I find that it reads weirdly. I'm also not sure it belongs in the privacy policy part.

Adimote replied:
fixed in 3a8f4b3

Adimote committed:
remove bad make call on `make watch`

Adimote committed:
remove mention of 'current and future sponsors'

Adimote committed:
restrict the future events to be 'related' events

Still want to keep the 'future events' section, because we want to
be able to use photos from this competition to promote the summer
school.

Adimote committed:
re-order privacy policy to be before media consent

This is so it doesn't confuse people when they're signing the
bottom on which part they're agreeing to.

Adimote committed:
make spacing consistent

Adimote committed:
Clarify that the forms are stored by SourceBots

and remove metioning of it being stored at the University of
Southampton, because it most likely won't be. It will probably be
stored in an encrypted file in a Google Drive somewhere.

Adimote committed:
change the minimum age to 16

Adimote committed:
s/your recordings/the recordings

Adimote committed:
s/events held by/activites of/

Adimote committed:
clarify: only applies to recordings at the event

Adimote committed:
move section on public internet storage

Move 'All recordings will be openly available to the internet' to
the media consent section, and reword it to make more sense.

trickeydan commented on form.tex:
```diff
@@ -0,0 +1,92 @@
+\textbf{For those aged under 12:}
+
+By signing below, I, the individual responsible for the above-named person, have read and understood the terms above, and grant my consent.\\
+Full Name: \vbox{\hbox{\TextField[width=5cm]{\null}}\kern2pt\hrule}\\
```
This seems slightly ambiguous. Perhaps `Name of Parent / Guardian` or similar would be more appropriate?

Adimote replied:
fixed in 9f7482c

Adimote committed:
remove the horrible red boxes

Adimote committed:
clarify parent/guardian signs the box

RealOrangeOne commented on form.tex:
```diff
@@ -0,0 +1,90 @@
+\begin{document}
+
+\newcommand{\compyear}{2018\xspace}
+\newcommand{\competition}{SourceBots \compyear}
```
As the rest of the media consent for is generic to an event, perhaps changing these variables to also just be `event` and `eventyear`? (I've looked, LaTeX doesnt have anything exceptionally nice for date formatting :sob: )

Adimote replied:
fixed in 916328f61419a2e66d9479a5bd9ba6b5f772e49b

PeterJCLaw commented on Makefile:
```diff
@@ -0,0 +1,34 @@
+	while true; do \
+		inotifywait --event CLOSE_WRITE -q -r *.tex; \
+			make; \
+		make; \
```
Why is there a double `make` here? (Also why is one of them extra indented?)

Adimote replied:
because when I was fixing the previous error I was being an idiot. fixed now. 48d519d

PeterJCLaw commented on form.tex:
```diff
@@ -0,0 +1,90 @@
+\vspace{0.3cm}
+\textbf{\large Media Consent}\\
+
+By signing this release form, you (the individual mentioned below) consent to your image, voice, biographical material, or other information being recorded in the duration of the \competition event by:
```
> being recorded in the duration of the SourceBots 2018 event by:

This reads really oddly to me. How about:

> being recorded at the SourceBots 2018 competition by:

Or perhaps (if we want 'event' in there to clarify that it's the competition weekend):

> being recorded at the SourceBots 2018 competition event by:

I suggest that we wrapper the "SourceBots 2018 competition event" portion into an `\event` command; I don't feel particularly strongly about this part, nor how that command is constructed.

Adimote replied:
fixed in ef25480c6114edb9c048465023d2438960e11426 and I fixed the event naming in 8e77f7f6f1836b1ab907e7ee5aeb07fc458abf8f

Adimote committed:
rename comp* commands to event*

Adimote committed:
fix silly mistake in makefile

Adimote committed:
fix readability of 'recorded at the event' section

Adimote committed:
rename event to 'SourceBots 2018 competition'

PeterJCLaw reviewed (commented):
Looks good (I've not checked that this latest version builds, but I trust you've done that)

PeterJCLaw reviewed (approved):
Looks good (I've not checked that this latest version builds, but I trust you've done that)
